src/models/linear_network.py

```diff
@@ -19,8 +19,10 @@ class LinearNetwork(ModelBase):
     def __init__(self, layer_sizes: Union[int, List[int]],
                  dropout: float = 0.25,
                  data_folder: Path = Path('data'),
+                 experiment: str = 'one_month_forecast',
```

Comment:
This should be a class attribute (i.e. added to the `__init__` function) since it will also be used by the predict function.

It should also be added to the predict function

Reply:
I added to the `base` model because every model will now have to have an experiment string to define where it is reading the data from